<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FOXSISIM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>531</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Module</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="doubleSpinBox">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>999999.999900000053458</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>310</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>999999.999900000053458</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Length of each segment</string>
      </property>
      <property name="text">
       <string>Segment Length</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Focal length, measured in centimeters from the center of the module</string>
      </property>
      <property name="text">
       <string>Focal Length</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>250</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Autocalculate the angle given a radius (and vice versa)</string>
      </property>
      <property name="text">
       <string>Autocalculate Dimensions</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>340</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>-999999.999900000053458</double>
      </property>
      <property name="maximum">
       <double>999999.999900000053458</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Offset of the detector from the focal point. A positive value means the detector is farther from the module.</string>
      </property>
      <property name="text">
       <string>Detector Offset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Width of the detector</string>
      </property>
      <property name="text">
       <string>Detector Width</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>400</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Height of the detector</string>
      </property>
      <property name="text">
       <string>Detector Height</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>370</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>999999.999900000053458</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>400</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>999999.999900000053458</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>502</width>
        <height>231</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>250</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <column>
       <property name="text">
        <string>Shell Radius</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Shell Angle</string>
       </property>
      </column>
     </widget>
     <widget class="QToolButton" name="toolButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add a row to the radii/angles table</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>250</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Remove a row from the radii/angles table</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Pixel resolution of detector</string>
      </property>
      <property name="text">
       <string>Detector Reso</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>430</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>430</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Move current row up</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>250</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Move current row down</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Plot a cross section of the module</string>
      </property>
      <property name="text">
       <string>Plot Cross Section</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="toolTip">
      <string/>
     </property>
     <attribute name="title">
      <string>Sources</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>500</width>
        <height>171</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>83</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>83</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Center</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Normal</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Width</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Height</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
       <property name="toolTip">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QToolButton" name="toolButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Add row to the sources table</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Remove row from sources table</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Autocalculate as many source settings as is possible</string>
      </property>
      <property name="text">
       <string>Autocalculate Dimensions</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Move current row up</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::UpArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_8">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>24</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Move current row down</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="arrowType">
       <enum>Qt::DownArrow</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Simulation</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>281</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Simulate</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Reset session, deleting all simulation variables</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Number of rays to simulate for each source</string>
       </property>
       <property name="text">
        <string>Rays Per Source</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>131</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Total rays to be simulated</string>
       </property>
       <property name="text">
        <string>Rays to Simulate</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Total rays simulated thus far</string>
       </property>
       <property name="text">
        <string>Total Simulated</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Start simulation. Once pressed, module/detector/source settings cannot be modified until session is reset.</string>
       </property>
       <property name="text">
        <string>Simulate</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>261</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Stop current simulation</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>10</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Detector Pixel Plot</string>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Plot the image captured by the detector</string>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>110</y>
        <width>191</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Scatter Plot</string>
      </property>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Rays will be colored by how many times they bounce in the module: none (red), once (green), twice (blue), more (black)</string>
       </property>
       <property name="text">
        <string>Color Bounce Count</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Generate a scatter plot of captured rays from one or more sources</string>
       </property>
       <property name="text">
        <string>Plot</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>111</height>
        </rect>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::MultiSelection</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>549</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_FOXSISIM_2"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_FOXSISIM">
   <property name="text">
    <string>About FOXSISIM</string>
   </property>
  </action>
  <action name="actionAbout_FOXSISIM_2">
   <property name="text">
    <string>About FOXSISIM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
